frontend/www/js/omegaup/components/course/Edit.vue

```diff
@@ -100,12 +104,35 @@
         <omegaup-course-assignment-list
           v-bind:assignments="data.course.assignments"
           v-bind:course-alias="data.course.alias"
+          v-bind:visibility-mode="visibilityMode"
+          v-on:emit-new="onNewAssignment"
+          v-on:emit-edit="(assignment) => onEditAssignment(assignment)"
+          v-on:emit-add-problems="(assignment) => onAddProblems(assignment)"
+          v-on:emit-delete="
+            (assignment) => $emit('delete-assignment', assignment)
+          "
+          v-on:emit-sort-homeworks="
+            (courseAlias, homeworksAliases) =>
+              $emit('sort-homeworks', courseAlias, homeworksAliases)
+          "
+          v-on:emit-sort-tests="
+            (courseAlias, testsAliases) =>
+              $emit('sort-tests', courseAlias, testsAliases)
+          "
         ></omegaup-course-assignment-list>
         <omegaup-course-assignment-details
-          v-if="data.selectedAssignment"
-          v-bind:show="false"
-          v-bind:update="true"
-          v-bind:assignment="data.selectedAssignment"
+          ref="assignment_details_list"
```

Comment:
nit:

```suggestion
          ref="assignment-details-list"
```

y se puede agregar un

```
@Ref('assignment-details-list') readonly assignmentDetailsList!: course_AssignmentDetails;
```

para evitar el cast explícito al momento de usarlo en L333 o L341?

Reply:
Listo